tensorrt_llm/_torch/disaggregation/nixl/agent.py

```
@@ -0,0 +1,253 @@
+                enable_prog_thread=use_prog_thread, backends=["UCX"], num_threads=num_workers
+            )
+            self.agent = nixl_agent(name, agent_config)
+
+        def register_memory(self, descs: RegMemoryDescs):
+            reg_descs = self.agent.get_reg_descs(descs.descs, descs.type)
+            self.agent.register_memory(reg_descs)
+
+        def deregister_memory(self, descs: RegMemoryDescs):
+            self.agent.deregister_memory(descs.descs, descs.type)
+
+        def load_remote_agent(self, name: str, agent_desc: bytes):
```

Ruff (0.14.10)

Unused method argument: `name`

(ARG002)